solr/solr-ref-guide/src/cloud-screens.adoc

```diff
@@ -23,7 +23,7 @@ This screen provides status information about each collection & node in your clu
 .Only Visible When using SolrCloud
 [NOTE]
 ====
-The "Cloud" menu option is only available on Solr instances running in <<getting-started-with-solrcloud.adoc#getting-started-with-solrcloud,SolrCloud mode>>. Single node or master/slave replication instances of Solr will not display this option.
+The "Cloud" menu option is only available on Solr instances running in <<getting-started-with-solrcloud.adoc#getting-started-with-solrcloud,SolrCloud mode>>. Single node or primary/secondary replication instances of Solr will not display this option.
```

Comment:
Should we say "legacy replication" instead? (especially if we decide to go with leader/follower)

Reply:
You are the only one truly **adamant** about changing to leader follower that I have seen thus far, so I'm not sure we will go that route. Unless, of course, you are willing to make the changes in that branch. You have a had start. :)

Reply:
[I'm not?](http://mail-archives.apache.org/mod_mbox/lucene-solr-user/202006.mbox/%3cCAKiERN77Q+LDCKMjhKcvitSLeY6PTikVnSErHv-Xa1RYsiqnrA@mail.gmail.com%3e). We can do a sweeping change after we are done with this PR and before merging. Shouldn't be too bad. Also, my comment is about the "mode", which has some other arguments also in that thread.

Reply:
Ok

Reply:
I'd never seen that email

Reply:
I don't think I'm on that list.